merp_custom_access_rights/static/description/index.html

```diff
@@ -0,0 +1,69 @@
+<section class="oe_container oe_dark">
+    <div class="oe_row">
+        <div class="oe_span12">
+            <h2 class="oe_slogan"><strong>Ventor</strong>/<strong>mERP</strong> Custom Access Rights module</h2>
+        </div>
+        <div class="oe_span12" style="text-align: center; margin-bottom: 5px">
+            <p style="padding: 5px; font-size: 16px; font-weight: bold; background-color: yellow; display: inline">
+                TO AVOID ANY ISSUES, PLEASE, USE ALWAYS LATEST VERSION FROM
+                <a href="https://github.com/ventor-tech/merp/tree/13.0">
+                    OUR GITHUB REPOSITORY
+                </a>
+            </p>
+        </div>
+        <div class="oe_span6">
+            <div class="oe_row_img oe_centered">
+                <img class="oe_picture oe_screenshot" src="merp_base.jpg">
+            </div>
+        </div>
+        <div class="oe_span6">
+            <p class="oe_mt32">
+Adds special access rights required by the "Ventor" app:
+            </p>
+            <ul>
+                <li>Warehouse Operations Menu</li>
+                <li>Inventory Adjustments Menu</li>
+                <li>One Location Inventory</li>
+                <li>Instant Inventory Menu</li>
+                <li>Internal Transfers Menu</li>
+                <li>Manufacturing Menu</li>
+                <li>Create Sales Order Menu</li>
+                <li>Quick Info Menu</li>
+                <li>Batch Picking Menu</li>
+                <li>Ventor Access Rights</li>
```

Comment:
Review description and check that here we are listing groups that really exists

Reply:
Checked, the one was deleted. The other are exists.